pkg/task/task.go

```diff
@@ -957,6 +989,17 @@ func (m *taskExecutor) getOEV1alpha1SPC() (err error) {
 	return
 }
 
+// getOEV1alpha1CSPC() will get the CStorPoolCluster as specified in the RunTask
+func (m *taskExecutor) getOEV1alpha1CSPC() (err error) {
+	cspc, err := m.getK8sClient().GetOEV1alpha1CSPCAsRaw(m.getTaskObjectName())
```

Comment:
same comments. Can we make use of cspc.KubeClient()

Reply:
something like -
```go
import cspc "github.com/openebs/maya/pkg/...../....."
```
```go
cspc, err := cspc.
		KubeClient().
		GetRaw(m.getTaskObjectName())
	if err != nil {
		return
	}
```